sim/priest/shadow/rotation.go

```diff
@@ -1,12 +1,13 @@
 package shadow
 
 import (
-	"time"
 	//"sort"
 	"github.com/wowsims/wotlk/sim/core"
 	"github.com/wowsims/wotlk/sim/core/proto"
 	"github.com/wowsims/wotlk/sim/core/stats"
+	"github.com/wowsims/wotlk/sim/priest"
 	"math"
+	"time"
```

Comment:
We keep the standard library imports separate from the github imports, with 1 blank line in between. So do:

```
math
time

github/...
github/...
```

Reply:
updated